pkg/cli/mirror/cincinnati_graph_image.go

```diff
@@ -1,6 +1,8 @@
 package mirror
 
 import (
+	"archive/tar"
```

Comment:
Seems like we vendor (and even [directly use](https://github.com/openshift/oc-mirror/blob/main/go.mod#L17)) https://github.com/mholt/archiver, maybe we just want to use that and not write our own `untar`?

Reply:
moved to mholt/archiver